src/main/resources/templates/thymeleaf/sicknote/sick_note.html

```diff
@@ -196,11 +196,14 @@ <h2 th:text="#{sicknote.progress.title}">Verlauf</h2>
             <tr th:each="comment, loopStatus : ${comments}">
               <td class="print:tw-hidden tw-text-blue-50 dark:tw-text-sky-800">
                 <img
-                  th:replace="fragments/avatar::avatar-bordered(url=${comment.person.gravatarURL + '?d=404&s=40'},niceName=${comment.person.niceName},width='40px',height='40px')"
-                  alt=""
+                    th:replace="fragments/avatar::avatar-bordered(url=${comment.person != null ? comment.person.gravatarURL + '?d=404&s=40' : '/unknown'},niceName=${comment.person != null ? comment.person.niceName : ''},width='40px',height='40px')"
```

Comment:
Open issue: url is set to 'unknown' this causes a bad request to
baseUrl+"/unknown" which leads to 404 and afterwards the fallback is
used. Not sure how to replace this without a failing call.

Setting url to '' or null causes a request to baseUrl which works and
does not cause fallback to avatar controller